modules/apps/foundation/roles/roles-admin-web/bnd.bnd

```diff
@@ -1,5 +1,6 @@
 Bundle-Name: Liferay Roles Admin Web
 Bundle-SymbolicName: com.liferay.roles.admin.web
 Bundle-Version: 1.0.10
+Export-Package: com.liferay.roles.admin.web.util
```

Comment:
We definitely should look for an alternative. If you export it then it's part of the public API, will affect versioning, we should keep it backwards compatible, ...

All things that I'm not sure we want to do for this class.